Arduino/MiP_ProMini_Pack_Library/src/mip.h

```diff
@@ -457,6 +456,8 @@ class MiP
     void readSoftwareVersion(MiPSoftwareVersion& software);
     void readHardwareInfo(MiPHardwareInfo& hardware);
 
+	void disconnectApp();
```

Comment:
I wonder if just ```disconnect()``` would be a better name for this new API?

Reply:
I have already written a disconnectBluetooth() which will be my next PR.  I think disconnectApp() will be an obvious differentiation and consistent naming convention compared to disconnectBluetooth(), however I'm happy to just call it disconnect() if you like.